src/pages/CandidateSearch/CandidateSearch.tsx

```diff
@@ -30,6 +30,23 @@ export const CandidateSearch: React.FC<{}> = (): React.ReactElement => {
     const [searchFilter, setSearchFilter] = React.useState<string>('name');
     const [openModal, setOpenModal] = React.useState<boolean>(false);
     const [applicantId, setApplicantId] = React.useState<number>();
+    const [candidates, setCandidates] = React.useState<IApplicantCard[]>();
+
+    React.useEffect(() => {
+        getCandidates();
+    },[]);
+
+    const getCandidates = async () => {
+        try {
+            const res = await fetch(`${process.env.REACT_APP_API_URL}/candidates`);
```

Comment:
Within the utils/constants file there is an object to store and manage all the api urls. 

If you add a new line in that you can reference the endpoint from here

Reply:
updated in latest PR